pilot/backup-restore.rst

```diff
@@ -0,0 +1,104 @@
+.. _backup-restore:
+
+Backup and Restore
+=======================================
+
+As :term:`RCC` is built upon AWS we're making use of the automated backup systems they provide.
+As such you are able to make use of these features in the event of data loss/corruption to restore from an alternate point in time.
+
+.. _restoring_machines:
+
+Restoring Machines
+---------------------------------------
+
+Due to the technical implementation automated backups show up under the packages section in Ronin, but it is not where they are restored from.
+
+
+.. image:: images/project_packages.png
+    :class: float-right
+    :scale: 50%
+
+As our backups are essentially packages, a restore is performed by `creating a new machine <https://blog.ronin.cloud/create-a-machine/>`_ and searching the "Project Packages" section under Step 1 of the machine creation screen.
```

Comment:
Use capital P for Packages or explicitly refer to them as 'Ronin Packages' throughout these docs to reduce ambiguity?

Reply:
I think 'Ronin Packages' would make more sense when it comes to end users knowing it's a Ronin specific thing.

```suggestion
Automated backups show up as Ronin Packages, a restore can be performed by `creating a new machine <https://blog.ronin.cloud/create-a-machine/>`_ and searching the "Project Packages" section under Step 1 of the machine creation screen.
```

Reply:
Should we use the term 'Project Packages' if that's what's shown in the UI?

Reply:
Also, where else in these docs do we need to make that change?  (probably best as a 2nd PR)